libs/vm-modules/include/vm_modules/ml/model/model.hpp

```diff
@@ -101,7 +101,7 @@ class VMModel : public fetch::vm::Object
 
   static void Bind(fetch::vm::Module &module);
 
-  ModelPtrType &GetModel();
+  void SetUnderlyingModelInstance(ModelPtrType const &instance);
```

Comment:
just SetModel is fine:

also why do you remove the getter?

Reply:
The only usage of this getter was in this file, to make assignments to its return value. This approach

a.getSomething() = b;

discards constness of the getter, and I guess it would be better to have explicitly non-constant setter.